bears/haskell/HaskellLintBear.py

```diff
@@ -21,7 +21,7 @@ class HaskellLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Duplication'}
-    CAN_FIX = {'Unused Code', 'Simplification'}
+    CAN_FIX = {'Unused Code', 'Redundancy'}
```

Comment:
I quote: ``suggests the replacement a $ b $ c with a . b $ c.`` i think this is an optimization